### PR TITLE
Add ipify metrics

### PR DESCRIPTION
**Pull request by edplato**
## Description

- [x] Add ipify ip capture for metrics

## Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/master/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

@nasa-gibs/worldview

### CODE REVIEW

File: web/js/app.js [hunk 1]
```diff
@@ -150,6 +150,16 @@ class App extends React.Component {
       }
       if (config.features.googleTagManager) {
         googleTagManager.init(config.features.googleTagManager.id); // Insert google tag manager
+
+        // get user ip address using https://www.ipify.org/ api
+        const response = await fetch('https://api.ipify.org?format=json');
+        const json = await response.json();
```
Comment (jasontk19) [inline web/js/app.js]: FWIW you shouldn't need to use the `await` keyword when calling this non-async function `response.json()`.  Similarly, it's not needed for accessing an object property: `json.ip`. Looks like it doesn't break anything though so, 👍

Comment (edplato) [reply web/js/app.js]: Thanks, for some reason I need it on the `response.json()` too or I keep getting `undefined`, but on `json.ip` it's definitely pointless.

Comment (jasontk19) [reply web/js/app.js]: Ah, yeah looks like I was thinking of some other library with a `response.json()` call that wasn't asynchronous.  Their docs clearly show `async response.json()`.  But now I'm wondering why 🤔

Comment (edplato) [reply web/js/app.js]: I based it off https://dev.to/johnpaulada/synchronous-fetch-with-asyncawait. Only thing I can think is the `response.json()` parsing could be a lengthy process if it's a lot of data so it needs to be a promise too?

**Pull request merged**